packages/GA4Client/src/commerce-handler.js

```diff
@@ -63,44 +68,38 @@ CommerceHandler.prototype.logCommerceEvent = function (event) {
         return logPromotionEvent(event);
     }
 
-    ga4CommerceEventParameters = buildParameters(event);
-
-    if (event.EventAttributes) {
-        ga4CommerceEventParameters = this.common.mergeObjects(
-            ga4CommerceEventParameters,
-            event.EventAttributes
-        );
-    }
```

Comment:
Reason for removal: This also gets overwritten by the next build function.  It is also duplicative of lines 123-126, which merge the new `ga4CommerceEventParameters` from each build function with custom event attributes.